vmdb/app/models/vm_redhat.rb

```diff
@@ -37,4 +37,12 @@ def self.calculate_power_state(raw_power_state)
     else             super
     end
   end
+
+  def supports_migrate?
+    false
+  end
+
+  def supports_reconfigure?
+    false
+  end
```

Comment:
It's also horrible performance because we have to instantiate all records to ask a question that the class knows how to answer.  :cry: